cms/templates/js/course-outline.underscore

```diff
@@ -4,6 +4,17 @@ var visibilityState = xblockInfo.get('visibility_state');
 var published = xblockInfo.get('published');
 var prereq = xblockInfo.get('prereq');
 var hasPartitionGroups = xblockInfo.get('has_partition_group_components');
+var userPartitions = xblockInfo.get('user_partitions');
```

Comment:
@bbaker6225 As we talked about, try adding the return value from get_visibility_partition_info to xblockInfo and use that in here instead of reproducing it in this file. xblockInfo is already a dumping ground of information... I don't feel bad about adding more.